src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -1137,11 +1305,16 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
                     sec_symmetry = self.m_create(Symmetry)
                     sec_symmetry.normalize(archive, logger)
 
+        print(self.model_system)
```

Comment:
Do these mean that you have tested this with your parser? If not how are you testing? And with normalization?